src/software/ai/passing/cost_function.cpp

```diff
@@ -335,27 +299,61 @@ double getStaticPositionQuality(const Field& field, const Point& position,
 }
 
 double calculateProximityRisk(const Point& point, const Team& enemy_team,
-                              double enemy_proximity_importance)
+                              const TbotsProto::PassingConfig& passing_config)
```

Comment:
Previously this function seemed backwards(?) where a lower score was given to areas with lots of enemies (start with risk of 1, and multiply it with a value <1 as more enemies where close to us). This seemed backwards since we would take `1 -` of the score returned by this function. Now the score starts at 0, and increases as more enemies are close to us. Once we take the `1 -` of it, we can multiply it with the other ratings we have where 1 is good, 0 is bad.

Reply:
we want to give the other team a chance against our top-tier AI